Makefile.am

```diff
@@ -813,6 +813,20 @@ endif
 
 check: $(CHECK_DEPS)
 
+install-all: install-kernel |  install-image |  install
+
+# Get clear containers image path
+IMAGE_PATH := $(shell dirname $(CONTAINERS_IMG))
+install-image:
+	curl https://raw.githubusercontent.com/clearcontainers/tests/master/.ci/install_clear_image.sh \
```

Comment:
I like the simplicity of this code, but this is a potential security risk; if the user is running `sudo make`, they are executing arbitrary shell code downloaded from a remote site as the superuser (`install_clear_kernel.sh` calls `rm -r` too ;)

I'm wondering if it makes sense to rework the two scripts in the `tests` repo slightly so that they have to be passed a parameter (`download`, or `install` or `show-install-locations`). That way, this `Makefile` could:

- Download the two scripts.
- Add a couple of rules (`download-kernel` and `download-image`).
- Ensure these two rules run as a non-`root` user:
  ```
  download-kernel:
  ifeq ($(shell id -u), 0)
    $(error ERROR: do not run this rule as root!)
  endif
  ...
  ```
- Make the new rules call the scripts with the `download` parameter.
- Change the `install-*` rules to run each script with the `show-install-locations` parameter. This would show 1 or more lines in the following format (which would map directory to `install(1)` arguments):
  ```
  <from> <to> <user> <group> <perms>
  ```
   ... where
   - `<from>` is the full path of the file to install,
  - `<to>` is the full path of the directory to install the file to.
- This `Makefile` could then call `install` itself (using `sudo`). This would be the only phase run as `root` so would limit the possibility of problems.

What do others think?